src/vivarium_census_prl_synth_pop/utilities.py

```diff
@@ -472,12 +471,19 @@ def write_to_disk(data: pd.DataFrame, path: Path):
         )
 
 
+def convert_objects_to_categories(data: pd.DataFrame) -> pd.DataFrame:
```

Comment:
I separated out this portion b/c vivarium is now writing out results inherently and so I use this function as the observation.results_formatter method.